drift/info_parser.py

```diff
@@ -26,7 +26,8 @@ def build_comparisons(systems_with_profiles, baselines):
     stripped_comparisons = [
         comparison
         for comparison in fact_comparisons
-        if comparison["name"] not in {"id", "system_profile_exists", "is_baseline"}
+        if comparison["name"]
+        not in {"name", "id", "system_profile_exists", "is_baseline"}
```

Comment:
Should these be indented or are they independent lines from the for loop and if block? Man, list comprehensions can be really hard to read

Reply:
its formatted like this from `black`. Lemme see if i can rewrite it, it's kinda grown over time.